log/log.go

```diff
@@ -86,26 +87,23 @@ func SetOutputFilename(logFile string) (*os.File, error) {
 	return filehandle, nil
 }
 
+// GetCrashInfoMsg returns the crash info message.
+func GetCrashInfoMsg() string {
+	msg := utils.Locale.Get("Please report this crash using %s", "GitHub Issues:")
+	msg += "\n" + "https://github.com/clearlinux/clr-installer/issues"
+	msg += "\n\n" + "Include the following as attachments to enable diagnosis:"
+	msg += "\n" + preConfName
+	msg += "\n" + logFileName
+	msg += "\n\n" + "You may need to remove any personal data of concern from the attachments."
```

Comment:
Should we not localize all of these sentences?

Reply:
Done.